src/protocols/ssh/settings.h

```diff
@@ -93,6 +93,12 @@ typedef struct guac_ssh_settings {
      */
     char* key_passphrase;
 
+    /**
+     * The public key, if any. If no public key is
+     * specified, this will be NULL.
+     */
```

Comment:
For consistency, can you please:
* Mention the base64-encoding (as above with `key_base64`
* Extended the length of the lines for the comment to the same length as the ones around it (roughly 80 characters).